requirements.txt

```diff
@@ -11,3 +11,4 @@ six>=1.10,<=1.99
 joblib~=0.11
 pubnub~=4
 cloudpickle~=0.2
+backports.weakref==1.0rc1 ; python_version < '3.4'
```

Comment:
Why does this need to be rc1? It looks like v1.0 was tagged on this project. And it looks like the version on PyPI is named "v1.0.post1".

Reply:
I think I copied this from their docs or something. Will fix.